### ISSUE TITLE
SPMT 0.4.1 crashes on every startup after update on MAC OSX High Sierra

### ISSUE THREAD

**Issue by supritdk** [PIVX-Project/PIVX-SPMT#7]
I have updated to most recent version but the app crashes on every start. Please let me know if there are any trouble shooting steps.

**Comment by random-zebra** [comment]
@supritdk can you please try to launch it from command line?
- open the Terminal.
- go inside the app folder:
```
cd SecurePivxMasternodeTool.app/Contents/MacOsX
```

- launch SPMT
```
./SecurePivxMasternodeTool
```

You could also try adding the flag `--clearAppData` (or the other flags described here https://github.com/PIVX-Project/PIVX-SPMT/blob/master/docs/features.md#features8)

**Comment by supritdk** [comment]
@random-zebra : I tried this. Although the app starts now, it crashes again within a few seconds after showing the spmt console. I tried with all the flags shown in the link that you have pasted.

**Comment by random-zebra** [comment]
Ok @supritdk 
Please check if this is fixed in v0.4.2.
